source/extensions/filters/network/zookeeper_proxy/decoder.h

```diff
@@ -80,7 +80,8 @@ class DecoderCallbacks {
   virtual ~DecoderCallbacks() = default;
 
   virtual void onDecodeError() PURE;
-  virtual void onRequestBytes(uint64_t bytes) PURE;
+  virtual void onDefaultRequestBytes(const OpCodes opcode, const uint64_t bytes) PURE;
```

Comment:
hard to understand the difference between `onDefaultRequestBytes` and `onRequestBytes`
either add some comment or changing the latter signature?

Reply:
I removed `onDefaultRequestBytes` and merged its code into `onRequestBytes`.